bHY6yh-d60082/cod-bqe7y7_bHY6yh-d60082.xml

```diff
@@ -0,0 +1,453 @@
+               <lb ed="#P"/><ref xml:id="bHY6yh-d60082-Rd1e161">Ibidem xii capitulo</ref> <quote xml:id="bHY6yh-d60082-Qd1e163">Qui operatur terram suam
+               <lb ed="#P"/>saturabitur panibus. Qui sectatur ocium 
+               <cb ed="#P" n="b"/>
+               <lb ed="#P"/>stultissimus est.</quote>
+            </p>
+            <p xml:id="bHY6yh-d60082-d1e171"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d60082-Rd1e173">Ibidem xiii capitulo</ref> <quote xml:id="bHY6yh-d60082-Qd1e173">vult et non vult piger
+               <lb ed="#P"/>Anima autem operantium impinguabitur.</quote>
+            </p>
+            <p xml:id="bHY6yh-d60082-d1e181"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d60082-Rd1e185">Ibidem capitulo xviiii</ref> <quote xml:id="bHY6yh-d60082-Qd1e183">pigredo immictit soporem
+               <lb ed="#P"/>anima dissoluta exuriet.</quote>
+            </p>
+            <p xml:id="bHY6yh-d60082-d1e189"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d60082-Rd1e195">Ibidem xx capitulo</ref> <quote xml:id="bHY6yh-d60082-Qd1e191">propter frigus piger arare no<lb ed="#P" break="no"/>luit
+               mendicabit ergo estate / et non 
+               <lb ed="#P"/>dabitur ei / Noli diligere somnum / ne te
+               <lb ed="#P"/>egestas opprimat a periculos tuis / et satura 
+               <lb ed="#P"/>te panibus.</quote>
+            </p>
+            <p xml:id="bHY6yh-d60082-d1e203"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d60082-Rd1e211">Ibidem xxi capitulo</ref> <quote xml:id="bHY6yh-d60082-Qd1e205">Cogitationes robusti semper 
+               <lb ed="#P"/>inabundantia omnis aut piger semper in 
+               <lb ed="#P"/>egestate est.</quote>
+            </p>
+            <p xml:id="bHY6yh-d60082-d1e213"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d60082-Rd1e223">Ibidem xxii capitulo</ref> <quote xml:id="bHY6yh-d60082-Qd1e215">vidisti virum velocem in
+               <lb ed="#P"/>opere suo coram regibus stabit et erit ante
+               <lb ed="#P"/>ignobiles.</quote>
+            </p>
+            <p xml:id="bHY6yh-d60082-d1e224"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d60082-Rd1e236">Ibidem xxvii capitulo</ref> <quote xml:id="bHY6yh-d60082-Qd1e226">Per agrum hominis pigri tran<lb ed="#P" break="no"/>siui 
+               et per vineam viri stulti / et ecce totum 
+               <lb ed="#P"/>inpleuerant vertice operuerant superficiem 
+               <lb ed="#P"/>eius spine / et materia lapidum destructa erat
+               <lb ed="#P"/>Quod cum vidissem posui in corde meo et
+               <lb ed="#P"/>exemplo didici disciplinam.</quote>
+            </p>
+            <!--<p xml:id="bHY6yh-d60082-d1e240"> 
+             <ref xml:id="bHY6yh-d60082-Rd1e254">28. c.</ref> <quote xml:id="bHY6yh-d60082-Qd1e242">Qui
+        operatur terram suam saturabitur panibus
+        Qui sectatur ocium replebitur egestate.</quote>
+            </p>-->
```

Comment:
Have you started commenting out paragraphs that aren't present instead of writing "omitted" between the `<p>` tags or something else going on here.

Reply:
I see an "omitted" below, so maybe this was already commented out for some reason??